server/src/main/java/org/apache/iotdb/db/mpp/execution/operator/exception/UnSupportedEventAggregationWindowTypeException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.mpp.execution.operator.exception;
+
+import org.apache.iotdb.db.mpp.execution.operator.window.WindowType;
+
+public class UnSupportedEventAggregationWindowTypeException extends RuntimeException {
```

Comment:
You can use `IllegalArgumentException` directly, we will check for correct semantics in the Analyzer phase.

Reply:
I've changed them to IllegalArgumentException.